plugins/file-open-save-new/core/src/main/javascript/app/services/data.service.js

```diff
@@ -73,8 +73,20 @@ define(
          *
          * @return {Promise} - a promise resolved once data is returned
          */
-        function getDirectoryTree(filter) {
-          return _httpGet([baseUrl, "loadDirectoryTree", filter].join("/"));
+        function getDirectoryTree(filter, connectionTypes) {
+          var method = "GET";
+          var url = [baseUrl, "loadDirectoryTree", filter].join("/");
+
+          var options = {
+            method: method,
+            url: _cacheBust(url),
+            headers: {
+              Accept: "application/json"
+            },
+            params: {connectionTypes: connectionTypes}
+          };
+          return $http(options);
```

Comment:
This appears to be another implementation of _wrapHttp with the addition of params. Can we add another helper method?

Reply:
Maybe so, let me take a stab at it in another PR